File: app/src/main/bilbil_api/dto/ChatRoomListDTO.java
package com.longtoast.bilbil_api.dto;

import lombok.Builder;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@Builder
public class ChatRoomListDTO {

    private Integer roomId;
    private LocalDateTime lastMessageTime;

    private Integer partnerId;
    private String partnerNickname;
    private String partnerProfileImageUrl;

    private Integer itemId;
    private String itemTitle;
    private String itemMainImageUrl;
    private int itemPrice;

    private String lastMessageContent;

    // ⭐ 추가
    private int unreadCount;
}

File: app/src/main/java/com/longtoast/bilbil/dto/ChatRoomListDTO.kt
// com.longtoast.bilbil.dto/ChatRoomListDTO.kt (안드로이드)

package com.longtoast.bilbil.dto

data class ChatRoomListDTO(
    val roomId: Int?,
    val lastMessageTime: String?,

    val partnerId: Int?,
    val partnerNickname: String?,
    val partnerProfileImageUrl: String?,

    val itemId: Int?,
    val itemTitle: String?,
    val itemMainImageUrl: String?,
    val itemPrice: Int?,

    val lastMessageContent: String?,

    val unreadCount: Int? // ★ 추가됨
)